src/BenchmarkDotNet/Attributes/HighPerformancePowerPlanAttribute.cs

```diff
@@ -0,0 +1,24 @@
+﻿using BenchmarkDotNet.Configs;
+using JetBrains.Annotations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BenchmarkDotNet.Attributes
+{
+    /// <summary>
+    /// determines if BenchmarDotNet changes power plan to High Performance
+    /// </summary>
+    [PublicAPI]
+    public class HighPerformancePowerPlanAttribute : Attribute, IConfigSource
```

Comment:
It makes sense to add `[AttributeUsage(AttributeTargets.Class | AttributeTargets.Assembly, AllowMultiple = true)]` here. It will allow comparing enabled/disabled configurations.